templates/reminder-no-maintainer.mail

```diff
@@ -0,0 +1,28 @@
+Dear port maintainers,
+
+The portscout new distfile checker has detected that one or more unmaintained 
+ports appear to be out of date. Please take the opportunity to check each of the 
+ports listed below, and if possible and appropriate, submit/commit an update. 
+Please consider also adopting this port. If any ports have already been updated, 
+you can safely ignore the entry.
+
+An e-mail will not be sent again for any of the port/version combinations
+below.
```

Comment:
`Only one update notification is sent for each port/version combination below:`

If this is just a copy from the original template, let's leave and separate any improvements into a separate PR, updating both.

Reply:
It is.